tailwind.config.ts

```diff
@@ -115,15 +118,36 @@ export default {
           from: { height: "var(--radix-collapsible-content-height)" },
           to: { height: "0" },
         },
+        aurora: {
```

Comment:
We can preferring not to make changes to tailwindcss config. Please move these animation to the component code.

You can refer to other components code for the approach.

Reply:
Done